gitreceive/receiver/flynn-receive.go

```diff
@@ -189,23 +199,25 @@ Options:
 			Processes: map[string]int{"web": 1},
 		}
 
-		watcher, err := client.WatchJobEvents(app.ID, release.ID)
-		if err != nil {
-			log.Fatalln("Error streaming job events", err)
-			return
-		}
-		defer watcher.Close()
```

Comment:
Yes the parameter sounds good, leave the timeout I think we should deal with that in a separate PR.